buildmaster/dataset_names.yml

```diff
@@ -0,0 +1,4 @@
+ATLAS_1JET_8TEV_R06_PTY: ATLAS_1JET_8TEV_R06
```

Comment:
I think the solution is to swap to old:new instead of new:old

So please, change `dataset_names.yml` to be something like this:

```yaml
ATLAS_1JET_8TEV_R06_DEC:
    dataset: ATLAS_1JET_8TEV_R06_PTY
    variant: decorrelated
CMS_1JET_8TEV: CMS_1JET_8TEV_PTY
```

For simplicity, when no variant is to be used, we can leave it simply as a mapping. This will allow to map many old datasets to a single new (plus its variant).
This will facilitate later on applying cuts depending on the variant if needed.